### ISSUE TITLE
Ability to disable auto image discovery

### ISSUE THREAD

**Issue by gacek1123** [harlan-zw/nuxt-simple-sitemap#24]
### Clear and concise description of the problem

**Google Search Console** is reporting issues with the `<image:image>` tags in my sitemap. I don't really need them anyways, so I would like the ability to disable them.

### Suggested solution

Add a new config option to allow users to disable this feature.

**Comment by harlan-zw** [comment]
Hey @gacek1123, thanks for the issue.

I can add support to disable them, as a workaround you should be able to use the hooks.

Would you mind providing any more details on the GSC issue? If there is a problem that may affect other users there may be a bug fix needed as well

**Comment by anirudhmurali** [comment]
Same issue, any way to disable images in the sitemap? Search Console isn't validating this sitemap despite having a structured XML.

![image](https://user-images.githubusercontent.com/3096325/229020899-16be40e3-6b32-46a3-9109-31350e10f54c.png)

**Comment by gacek1123** [comment]
@harlan-zw Google is showing 2 different errors:
- first one saying that `image` has an invalid child tag `url`
- second one saying that `image` doesn't have a required tag `loc`

So it seems that it's actually just a bug

**Comment by anirudhmurali** [comment]
Any fixes for this or tutorial to remove images with hooks? Unable to get the sitemap submitted.

**Comment by harlan-zw** [comment]
Thank you for clarifying and apologies for the delay.

 I've pushed up a fix in 2.3.0 that should have images validate correctly.

If you'd still prefer to have the images disabled I've also added the option:

```ts
export default defineNuxtConfig({
  sitemap: {
    // avoids images being added to sitemap
    discoverImages: false
  }
})
```

Let me know if you have any feedback on it

**Comment by yeganathan18** [comment]
Thanks @harlan-zw it worked now 🎉
